### ISSUE TITLE
Update dependencies

### ISSUE THREAD

**Issue by ffont** [ffont/freesound-explorer#34]
We have updated dependencies, nooooo :(

**Comment by giuband** [comment]
This will imply some rewriting in the `mapStateToProps`

**Comment by giuband** [comment]
I would suggest to migrate to `create-react-app` to make this repo easier to maintain up-to-date :) I could try to do it in the next few days.

**Comment by ffont** [comment]
"easier ti maintain" -> sounds great!

**Comment by ffont** [comment]
Thanks @noVaSon! I already merged the changes. Are other dependencies still outdated?

**Comment by noVaSon** [comment]
Yes, there were depreciation warnigns about calling PropTypes from React main package.
I already applied the supposed updates in my master branch. I'll think about how to transfer my changes in an understandable fashion..

**Comment by noVaSon** [comment]
When all current Pull Requests are reviewed and merged - I think it is time to do an npm prune .
I tried it and it deletes over 100 unused dependencies!
What a feeling ^^

**Comment by carthach** [comment]
These seem to be out of date again.. any plans to update?

**Comment by noVaSon** [comment]
Hi @carthach!
This one was lost, but I am on it right now! There is some heavy work to do, we hat more than 2k vulnerabilies! Frightening.
Best, Eric

**Comment by carthach** [comment]
Better late than never! Cheers for replying Eric!
